### DISCUSSION TITLE
No Setup Guide?

### DISCUSSION THREAD

**Discussion by molikd** [hyperspike/hyperctl#2] [Ideas] [answered]
This boils down to: please add a short setup guide. 

While I think you should eventually turn over to a github hosted webpage for documentation of hyperspike, a short description of setup process, assumably a docker build command, would be useful. Most packages young put this in the README.md, and with maturation, build out either their included github wiki, or host a simple jekyll site for documentation. 

It should cover a initial build and one or two more steps that you might want to do with hyperspike. 

For all of the problems with their code, ncbi actually has software documentation content down: https://github.com/ncbi/blast_plus_docs 

This should not be confused with code documentation, this would full more on the user documentation of the spectrum. 

-Dave

**Comment by dmolik** [comment, marked as answer]
I'll slide a setup guide into the readme today.

I'm still working on the documentation site: https://docs.hyperspike.io
